spec/jobs/update_category_stats_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
```suggestion
```

This line is no longer necessary since we [auto require it](https://github.com/discourse/discourse/blob/e9f016726ae02084800ad45ca85062f35756affe/.rspec#L1).